server/src/controllers/users/signUp.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from 'express';
+import bcrypt from 'bcrypt';
+import signupValidation from '../validation';
+import { User } from '../../database/models';
+import { jwtSign } from '../../utilities';
+
+const signUp = async (req: Request, res: Response, next: NextFunction) => {
+  try {
+    signupValidation(req.body).then(async () => {
+      const { email, password, confirmPassword } = req.body;
+      const isEmailExist = await User.findOne({ where: { email } });
+      if (isEmailExist) {
+        res.status(400).json({ success: false, message: 'Email is already exist !' });
+      } else if (confirmPassword === password) {
+        bcrypt.hash(password, 10).then(async (encryptedPass) => {
+          // eslint-disable-next-line max-len
+          const user = await User.create({ ...req.body, password: encryptedPass, confirmPassword: encryptedPass });
```

Comment:
you don't have to store confirmPassword in the database it's just for validation